components/alert/w-alert.vue

```diff
@@ -21,29 +23,18 @@ const props = defineProps({
   ...modelProps(),
 });
 
-const possibleTypeProps = ["negative", "positive", "warning", "info"];
+const possibleTypeProps = ['negative', 'positive', 'warning', 'info'];
 
-const emit = defineEmits(["update:modelValue"]);
+const emit = defineEmits(['update:modelValue']);
 const model = createModel({ props, emit });
 const alertColorType = computed(() => possibleTypeProps.find((e) => props[e]));
 const activeWrapperClassNames = computed(() => ccAlert[alertColorType.value]);
-const activeIconClassNames = computed(
-  () => ccAlert[`${alertColorType.value}Icon`]
-);
-const wrapperClass = computed(() => [
-  ccAlert.alert,
-  activeWrapperClassNames.value,
-]);
+const activeIconClassNames = computed(() => ccAlert[`${alertColorType.value}Icon`]);
+const wrapperClass = computed(() => [ccAlert.alert, activeWrapperClassNames.value]);
 const iconClass = computed(() => [ccAlert.icon, activeIconClassNames.value]);
 
 const iconComponent = computed(() =>
-  props.negative
-    ? IconError16
-    : props.positive
-    ? IconSuccess16
-    : props.warning
-    ? IconWarning16
-    : IconInfo16
+  props.negative ? IconError16 : props.positive ? IconSuccess16 : props.warning ? IconWarning16 : IconInfo16,
```

Comment:
Can we do something about this? 🙈 
(...or maybe this is correct and it would have been better to have it as a switch to start with.)

Reply:
Hmm.... and why did it add a trailing comma? 😅  That's just plain wrong.

Reply:
Yeah I agree that doesn't look nice! 🙈 I think that is coming from the `vue-eslint-parser`. Let me see if I can add a rule for that! 🤔

Reply:
I personally hate nested ternary statements like these. Over time they just become impossible to read. I know the alternative to this is a plain old if else block, but atleast that maintains readability:

`
let icon;
if (props.negative) {
  icon = IconError16;
} else if (props.positive) {
  icon = IconSuccess16;
} else if (props.warning) {
  icon = IconWarning16;
} else {
  icon = IconInfo16;
}
`

Reply:
I don't think changing `printWidth` to `100` is the solution. We might still see other examples of code that don't wrap though we'd visually expect them to. I'd also consider replacing a nested ternary statement with something a bit more readable at this stage 😄  As for the other rules not applying, not sure what's wrong but we could perhaps look into it together to better understand the problem 😊

Reply:
I think I rambled on a bit too much in my previous comment. 🙈  What I meant before was that when trying to override only the `printWidth` from the `@warp-ds/eslint-config`, it overrode all of the other rules too. But I won't add this to the `.eslintrc-file`, because I don't think we want to override our `@warp-ds/eslint-config`. :) So, I'll just rewrite this to an if-else instead of a nested ternary statement :)